test/test-clusters/src/main/java/org/elasticsearch/test/cluster/local/LocalClusterSpec.java

```diff
@@ -194,6 +194,10 @@ public boolean isMasterEligible() {
             return getSetting("node.roles", "master").contains("master");
         }
 
+        public boolean hasRole(String role) {
+            return getSetting("node.roles", "[]").contains("search");
```

Comment:
It seems this should be `.contains(role);`

Reply:
🤦